AnnService/inc/Core/SPANN/ExtraFullGraphSearcher.h

```diff
@@ -602,14 +799,51 @@ namespace SPTAG
                     }
                 }
 
+                if (ptr->ReadBinary(sizeof(m_enableDeltaEncoding), reinterpret_cast<char *>(&m_enableDeltaEncoding)) != sizeof(m_enableDeltaEncoding))
```

Comment:
Does it mean we will not be able to load SSD index built by old version of SPTAG, since the head info will not contain these config bytes? 

I don't think we can break compatibility like this

Reply:
> Does it mean we will not be able to load SSD index built by old version of SPTAG, since the head info will not contain these config bytes?
> 
> I don't think we can break compatibility like this

Another approach is to add these parameters in `m_extraSearcher->SearchIndex` function, with which we may have to change the Nuget package interface. Which approach do we prefer?

Reply:
Fixed by adding options instead of dumping configurations to head info. The head info format will only be changed when compression enabled.